src/test/java/uk/gov/justice/digital/delius/controller/secure/CustodyKeyDatesControllerTest.java

```diff
@@ -77,9 +85,9 @@ void WillReturn404WhenOffenderNotFound() {
         }
 
         @Test
-        void WillReturn404WhenConvictionForBookingNotFound() throws ConvictionService.DuplicateActiveCustodialConvictionsException {
+        void WillReturn404WhenConvictionForBookingNotFound() {
             when(offenderService.offenderIdOfNomsNumber(any())).thenReturn(Optional.of(99L));
-            when(convictionService.getSingleActiveConvictionIdByOffenderIdAndPrisonBookingNumber(any(), any())).thenReturn(Optional.empty());
```

Comment:
Not massively important but could consider using the actual parameter values in the "when". (99L and 44463B)